### ISSUE TITLE
No way of building it on macOS 10.11 (Xcode 8.x)

### ISSUE THREAD

**Issue by dmilith** [slash-hq/slash#23]
I installed Swift 4.2 pkg from their page but Xcode still fails to build Slash on 10.11.6 :(

**Comment by dmilith** [comment]
Can you maybe provide a prebuilt version which will work on 10.11 please? :)

**Comment by dmilith** [comment]
Regardless, I did build which works on 10.11 as well from master - http://s.verknowsys.com/80ec5904be5a302bb466dbc7ae4e479e (It's prebuilt binary that works on 10.11+)

**Comment by philippkeller** [comment]
Same issue here. Getting:

```
The following build commands failed:
        CompileSwift normal x86_64
        CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(2 failures)
```

@dmilith : the link you posted is a 404, can you provide that binary somehow or describe how you compiled it?

**Comment by glock45** [comment]
@philippkeller Hmmm....,  Xcode (Version 10.1 (10B61)) works for me. Could you please provide more info about the error?

You can find it here:
<img width="642" alt="zrzut ekranu 2019-02-05 o 23 20 37" src="https://user-images.githubusercontent.com/492449/52305051-d6639c00-299c-11e9-8619-091131c50573.png">

**Comment by philippkeller** [comment]
oh, it now worked after doing

```
git clone git@github.com:slash-hq/slash.git
cd slash
xcodebuild
```

Seems that the compilation via brew install is broken

**Comment by dmilith** [comment]
Yes it works from 10.14, but not from 10.11. Cause some magical reasons Xcode has only SDK 10.12 and build fails anyway http://s.verknowsys.com/83a21b7cfd8c3db8903a370f4748910f.png
